app/src/test/java/com/onboarding/wheatherapp/util/WeatherTestUtils.kt

```diff
@@ -0,0 +1,45 @@
+package com.onboarding.wheatherapp.util
+
+import com.onboarding.wheatherapp.data.entity.Data
+import com.onboarding.wheatherapp.data.entity.Main
+import com.onboarding.wheatherapp.data.entity.WeatherDescription
+import io.reactivex.rxjava3.core.Scheduler
+import io.reactivex.rxjava3.disposables.Disposable
+import io.reactivex.rxjava3.internal.schedulers.ExecutorScheduler
+import java.util.Observable
+import java.util.concurrent.Executor
+import java.util.concurrent.TimeUnit
+
+object WeatherTestUtils {
+
+    const val ZERO = 0
+    const val TEMP = "10"
+    const val TEMP_MIN = "11"
+    const val TEMP_MAX = "12"
+    const val ICON_WEATHER_STRING = "HOLA"
+    const val date_1 = "10/02/2021 16:39:00"
+    const val date_2 = "10/02/2021 17:39:00"
+    const val date_3 = "10/02/2021 12:00:00"
+    val main = Main(TEMP, TEMP_MIN, TEMP_MAX)
+    val weatherDescription = WeatherDescription(ICON_WEATHER_STRING)
+    val listWeather: MutableList<WeatherDescription> = mutableListOf(weatherDescription)
+    val data_1 = Data(main, listWeather, date_1)
+    val data_2 = Data(main, listWeather, date_2)
+    val data_3 = Data(main, listWeather, date_3)
+    val forecastExtendTest: MutableList<Data> = mutableListOf(data_1, data_2, data_3)
+    val expectedFilteredList = listOf<Data>(data_3)
```

Comment:
why you are defining this val`s here?

Reply:
the configuration for each value, must stay in a private method on the test o in each method

Reply:
Solved!